releases/tex/master/extra/crowbar-backup

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+FILE="/root/crowbar_backup.tar.gz"
```

Comment:
Would be nice to have an option to specify the file where the backup will be put (check the old crowbar-backup script for this)